pytest.ini

```diff
@@ -3,6 +3,7 @@ addopts =
     --pyargs snowflakes.tests
     --pyargs snovault.tests
     --pyargs snovault.elasticsearch.tests
+    # --pyargs snovault.elasticsearch.uuid_queue.tests
```

Comment:
Redis is not yet setup for testing, redis server needs to be running locally.  Leaving this commented out till fixed.